src/app/index.html

```diff
@@ -2,8 +2,76 @@
 </app-nav>
 
 <stache
- pageTitle="Stache 2"
- layout="container"
- showBreadcrumbs="false">
+  layout="container"
+  pageTitle="Stache 2">
+  <p>
+    Welcome to the SKY UX Stache template.  This template is dynamically cloned anytime <stache-code>skyux new --template stache</stache-code> is run.  For more information about SKY UX, visit the README.
+  </p>
 
+<stache-tutorial>
+  <stache-tutorial-step showNumber="false">
```

Comment:
I like that the `showNumber` key works no matter where you put the step. One thing that would be cool is if we could also specify a character to appear in that number circle on non-numbered steps. For example, the github pages pattern has a question mark in there when user choice is required (pick an option). But this might be something that we do in the future rather than with this PR.